languages/elixir/exercises/concept/agents/.docs/introduction.md

```diff
@@ -0,0 +1,23 @@
+The `Agent` module facilitates an abstraction for spawning processes and the _receive_, _send_ loop -- from here, we will call processes started using the `Agent` module _'agent processes'_. An _agent process_ might be chosen to represent a central shared state.
+
+To start an unlinked _agent process_, `Agent` provides the `start/2` function:
+
+```elixir
+# Start an agent process with an initial value of an empty list
```

Comment:
This comment may lead people to believe they should always start an agent process with an empty list. An explanation of what that argument is could be useful, maybe before the code snippet/